docs/user-guide/quick-start.md

```diff
@@ -22,7 +22,7 @@ $ kubectl run my-nginx --image=nginx --replicas=2 --port=80
 deployment "my-nginx" created
 ```
 
-To expose your service to the public internet, run:
+To expose your service to the public internet, run the following. Note, the type, LoadBalancer, is highly dependant upon the underlying platform that Kubernetes is running on. Type LoadBalancer may work in public cloud environments just fine but may require some additional configuration in a private cloud environment (ie. OpenStack). 
```

Comment:
Nit: please refrain from mentioning specific cloudproviders if you can, since that's bound to drift from implementation. 

Also suggest including something like:
```
If your clourprovider doesn't have a loadbalancer implementation for Kubernetes, you can simply use 
the allocated [nodePort](link to nodeport service) as a rudimentary form of round robing loadblancing 
across your endpoints.
```